### ISSUE TITLE
Roundstart bottles of mannitol pills contain enough mannitol per pill to easily overdose.

### ISSUE THREAD

**Issue by newfren** [tgstation/tgstation#61921]
observed over multiple rounds

each pill in the bottle is 50u, the overdose on mannitol is 15u.

**Comment by Mooshimi** [comment]
I don't think this is a bug. It's not exactly harmful if you overdose. Anyone else have some feedback?

**Comment by newfren** [comment]
it seems to me like it was an oversight from when mannitol did not have an overdose. 

just seems very strange that nt would provide to each of its stations pills that are like 3 times larger than they would sensibly be.
